dipy/workflows/denoise.py

```diff
@@ -103,13 +102,14 @@ def run(self, input_files, bvalues_files, bvectors_files, sigma=0,
             Default 0: it means sigma value estimation with the Manjon2013
             algorithm [3]_.
         b0_threshold : float, optional
-            Threshold used to find b=0 directions (default 0.0)
+            Threshold used to find b=0 directions
         bvecs_tol : float, optional
             Threshold used to check that norm(bvec) = 1 +/- bvecs_tol
-            b-vectors are unit vectors (default 0.01)
+            b-vectors are unit vectors.
         patch_radius : int, optional
             The radius of the local patch to be taken around each voxel (in
-            voxels). Default: 2 (denoise in blocks of 5x5x5 voxels).
+            voxels) For example, for a patch radius with value 2, the denoising
+            will take place in blocks of 5x5x5 voxels.
```

Comment:
I realize that since these methods can be applied to N-D images in theory, we should specify that the example applies to a 3D image, e.g.::
```
voxels). For example, for a patch radius with value 2, and assuming the input image is a 3D image, the denoising will take place in blocks of 5x5x5 voxels.
```

Sorry for not having realized before.

Reply:
Addressed in e5af17f